components/badges.css

```diff
@@ -0,0 +1,39 @@
+:root{
+    --primary: blueviolet;
+    --white: #fff;
+}
+.container-badge{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem 0;
+}
+.badges{
+    margin-right: 2rem;
+    position: relative;
+}
```

Comment:
What does the class .badges  do ?

Reply:
It's the parent container which hold the icon with the badge